server/controllers/finance/reports/break_even_fee_center/report.handlebars

```diff
@@ -0,0 +1,313 @@
+<!doctype html>
+<html>
+  {{> head title="TREE.BREAK_EVEN_FEE_CENTER_REPORT"}}
+  <body>
+  <div class="container">
+    {{> header}}
+
+    <!-- body -->
+    <div class="row">
+      <div class="col-xs-12">
+        <!-- page title  -->
+        <h3 class="text-center text-uppercase">
+          <strong>{{translate 'TREE.BREAK_EVEN_FEE_CENTER_REPORT'}}</strong>
+        </h3>
+
+        <h4 class="text-center">
+          <strong class="text-capitalize">{{date period.fiscalYearStart "MMMM YYYY"}}</strong>
+           -
+          <strong class="text-capitalize">{{date period.end_date "MMMM YYYY"}}</strong>
+        </h4>
+        {{#each principalSelected}}
+          <table style="page-break-after: always; width: 80%; margin-right: auto; margin-left: auto" class="table table-striped table-condensed table-report table-bordered">
+            <thead>
+              <tr style="background-color:#dcdcdc"> 
+                <th colspan="3" class="text-uppercase">
+                  <h5>
+                    <strong>
+                      {{ label }}
+                    </strong>
+                  </h5>
+                </th> 
+              </tr>
+            </thead>
+            <tbody>
+              <tr class="text-right" style="background-color:#efefef; width: 40%; font-size: 15px;">
+                <td colspan="2"> {{translate "TABLE.COLUMNS.NUMBER_CASES" }} </td>
+                <td> <strong> {{ numberCase }} </strong> </td>
+              </tr>
+              <tr>
+                <th style="width: 40%; font-size: 15px; text-align: center;" rowspan="3" scope="row">
+                  <strong>
+                    {{translate "TABLE.COLUMNS.CHARGES" }}
+                  </strong>                  
+                </th>
+                <td class="text-right" style="width: 40%; font-size: 15px">
+                  <i>
+                    {{translate "TABLE.COLUMNS.VARIABLE_CHARGES" }}
+                  </i>                  
+                </td>
+                <td class="text-right" style="width: 20%; font-size: 15px">
+                  {{debcred balanceVariableCost metadata.enterprise.currency_id}}
```

Comment:
Are you sure this `metadata.enterprise.currency_id` is respected?  I think it should be `../metadata.enterprise.currency_id`.  Try changing the currency and see if it is.

Reply:
It's possible, i need Vanga database to do this task well, if this problem is generalized in another report, maybe it will require the creation of an issue